commit 3f35614079f9ce795a0b449dcdf292157327a0e3 (refs/changes/36/3836/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-04-20T21:41:37+00:00 (1 year, 6 months ago)
    
    joyent/node-smartdc-auth#14 Agent errors result in useless exception
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/kr-agent.js b/lib/kr-agent.js
index 23442e7..53cef6e 100644
--- a/lib/kr-agent.js
+++ b/lib/kr-agent.js
@@ -135,7 +135,6 @@ AgentKeyPair.prototype.createSign = function (opts) {
 
 		self.akp_agent.sign(pub, data, function (err, sig) {
 			if (err) {
-				cache.put(ck, err);
 				cb(err);
 				return;
 			}
diff --git a/package.json b/package.json
index 6f3113c..d1db3fb 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "smartdc-auth",
     "description": "Joyent Authentication Library",
-    "version": "2.5.6",
+    "version": "2.5.7",
     "author": "Joyent (joyent.com)",
     "contributors": [
         "Mark Cavage",
